tests/test_invenio_assets_filters.py

```diff
@@ -75,16 +77,23 @@ class TestInvenioAssetsCleanCSSFilter(TempEnvironmentHelper):
         """
     }
 
-    def test_clean_CSS(self, static_dir):
+    def test_clean_CSS(self, static_dir, cleancss_version):
```

Comment:
Here I can't use `pytest.mark.skipif(cleancss_version > 4)` because it can't resolve `cleancss_version` as we are before the function